gender_novels/corpus_gen.py

```diff
@@ -7,25 +7,30 @@
 import pywikibot
 
 from gender_novels import common
-from gender_novels import novel
 
 # TODO: A lot of things
 
 GUTENBERG_MIRROR_PATH = ''
-GUTENBERG_METADATA_PATH = ''
+GUTENBERG_METADATA_PATH = Path('corpora', 'Gutenberg', 'Gutenberg.csv')
+metadata_list = ['id', 'author', 'date', 'title', 'country_publication', 'author_gender', 'subject', 'corpus_name',
```

Comment:
Not sure if we want to include `id` and `subject`. For ID, we may want to reference the LOC call number, for example. In terms of subject, I am not sure that that data is particularly useful especially since this information seems to be reductive — "Social life and customs" is not how I would describe *Persuasion*. If we also limit our work for this iteration to the Gutenberg collection, then including `corpus_name` will be redundant.

Reply:
I wasn't sure whether we were still doing other corpuses so that's why I had corpus_name.  The id will be needed for pulling the texts from the mirror later during analysis.  The subject here is not complete, but I do agree that the subject is somewhat vague and LoC subjects are often not exactly accurate.  I'll take it out unless the analysis team decides they want it.

Reply:
Cool, did not know that the `id` was needed for the mirror pulling. I think given time constraints we may not get to additional corpora but we can leave it in for now but may need to clean it later. I think taking out the `subject` makes sense especially if we are trying to appeal to humanities researchers who like nuance 😄

Reply:
Re corpus name: I think it's still useful to keep it to distinguish it from the `sample_novels` corpus.

Re subjects: I think they could be useful to at least decide if fiction or not. Is it possible to get all the subjects and put them in a list? There are more listed on Project Gutenberg: https://www.gutenberg.org/ebooks/105 

Would it be possible to add the LoC class (not the LoC call number)? That seems like the best short description of the book's category.

Reply:
Should be fairly easy to get all the Gutenberg subjects.  Haven't looked at LoC yet.  Should we keep all the current metadata fields then?

Reply:
I would say yes. They all seem reasonable. Though I'd rename `id` to `gutenberg_id` to make it clear what kind of id it is. 
Plus by using `id` as a variable, you're shadowing the `id()` function. Which you're unlikely to need but it's still not great practice.

Reply:
Final thought after looking over the file again: Can you standardize `id` and `novel_id` (both in the dictionary and as a function parameter) to be `gutenberg_id`?  (Unless there is a difference between id and novel_id) I'll then merge the commit.